plugins/haconfig/keepalived/helpers.go

```diff
@@ -0,0 +1,78 @@
+package keepalived
```

Comment:
plugins/ is an artifact of some older decisions, make this pkg/ipfailover/keepalived/...